### PR TITLE
Update Blog Page

### PR DESCRIPTION
**Pull request by DivyanshuParwal**
Implemented the following 
- Removed blog.module.css file (resolves one task of #28)
- Updated and used TopBanner component to implement hero image in place of the previously implemented way in the _blog.jsx_ file.
- Added blog link in blog button in the footer

### CODE REVIEW

File: src/components/TopBanner.jsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const TopBanner = ({ bannerImg, pageTitle, pageDesc }) => {
   return (
     <TopScreen>
       <Image
-        alt="Get Involved"
+        alt={bannerAlt ? bannerAlt : ''}
```
Comment (paulcretu) [inline src/components/TopBanner.jsx:80]:
```suggestion
        alt={bannerAlt ?? ''}
```

File: src/components/TopBanner.jsx [hunk 2]
```diff
@@ -107,7 +107,7 @@ const TopBanner = ({ bannerImg, pageTitle, pageDesc }) => {
               />
             </ScrollDownButton>
           </ScrollElement>
-          <PageDesc>{pageDesc}</PageDesc>
+          {pageDesc ? <PageDesc>{pageDesc}</PageDesc> : null}
```
Comment (paulcretu) [inline src/components/TopBanner.jsx:110]:
```suggestion
          {pageDesc && <PageDesc>{pageDesc}</PageDesc>}
```

**Review by paulcretu — APPROVED** [review]
Looks great! Just some small logic simplifications, but non-blocking (will let you take a look before I merge it though)

File: src/components/TopBanner.jsx (hunk 1 above)
Comment (ShwetKhatri2001) [reply src/components/TopBanner.jsx:80]: Do we require an extra prop as `bannerAlt` ? Can't we use `pageTitle` instead as the value for Image alt property ?

Comment (paulcretu) [reply src/components/TopBanner.jsx]: Good point @ShwetKhatri2001, you're probably right. I'm just going to merge this as-is for now, we can change that in a later PR

**Pull request merged**